openiddict/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.0.1
```

Comment:
This should be a minor feature upgrade (i.e `4.1.0`) as it's arguably a feature.

The version in the csproj will also need to update :)

Reply:
I have update the version in the changlog and csproj now.